app/models/following.rb

```diff
@@ -0,0 +1,9 @@
+class Following
```

Comment:
This feels like a better encapsulated/abstracted version instead of using one class.